Api/Data/NodeInterface.php

```diff
@@ -10,6 +10,7 @@ interface NodeInterface
     const MENU_ID = 'menu_id';
     const TYPE = 'type';
     const CONTENT = 'content';
+    const ENTITY = 'entity';
```

Comment:
This is currently not needed. It got left behind when experimented on adding the `getCategory()` method into the Node object instead of the Block object.

However, now that I noticed it, I think it *would* in fact be much better from the API user's point of view if the method was implemented into the Node object.

The reason I ended up adding the method to the block is that also the [getCategoryUrl()](https://github.com/SnowdogApps/magento2-menu/blob/master/Block/NodeType/Category.php#L134-L151) has been implemented in it.

Why is this? Why not just make it possible to call `$node->getUrl()`? And respectively `$node->getCategory()`?